### PR TITLE
Added feed.html and public-feed.js. Edited MessageFeedServlet.java.

### PR DESCRIPTION
**Pull request by sabrinakundu**
- Added feed.html to display the Message Feed webpage that shows messages from all users. 

- Added public-feed.js that includes functions to get messages data and build the message boxes.

- Changed doGet method in MessageFeedServlet.java to return a JSON representation of the Message data, which will be accessed by public-feed.js.

### CODE REVIEW

File: src/main/webapp/feed.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <script src="/js/public-feed.js"></script>
+  </head>
+  <body onload="buildUI()">
+    <div id="content">
+      <h1>Message Feed</h1>
+      <hr/>
+      <div id="message-container">Loading...</div>
+    </div>
+  </body>
+</html>
```
Comment (bandreasv) [inline src/main/webapp/feed.html:16]: Please add an empty line at the end of the file. It is good practice.

**Review by ldickeson — APPROVED** [review]
Everything looks good. My only comment is that maybe you could add in some comments telling what some chunks of code are doing. Other than that it looks fine

File: src/main/webapp/js/public-feed.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+   messageDiv.appendChild(headerDiv);
+   messageDiv.appendChild(bodyDiv);
+   
+   return messageDiv;
+  }
+  
+  /** Fetches data and populates the UI of the page. */
+  function buildUI(){
+   fetchMessages();
+  }
```
Comment (bandreasv) [inline src/main/webapp/js/public-feed.js:55]: Add a new line at the end of the file

File: src/main/webapp/feed.html (hunk 1 above)
Comment (sabrinakundu) [reply src/main/webapp/feed.html:16]: ok thanks!

File: src/main/webapp/js/public-feed.js (hunk 2 above)
Comment (sabrinakundu) [reply src/main/webapp/js/public-feed.js:55]: Yes, will do!

**Pull request merged**